books/free-programming-books-langs.md

```diff
@@ -2045,6 +2046,11 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [The Q# quantum programming language user guide](https://learn.microsoft.com/en-us/azure/quantum/user-guide) (HTML)
 
 
+### Qiskit
+
+* [Learn Quantum Computation using Qiskit](https://qiskit.org/textbook/preface.html) - Amira Abbas and Stina Andersson and Abraham Asfaw and Antonio Corcoles and Luciano Bello and Yael Ben-Haim and Mehdi Bozzo-Rey and Sergey Bravyi and Nicholas Bronn and Lauren Capelluto and Almudena Carrera Vazquez and Jack Ceroni and Richard Chen and Albert Frisch and Jay Gambetta and Shelly Garion and Leron Gil and Salvador De La Puente Gonzalez and Francis Harkins and Takashi Imamichi and Pavan Jayasinha and Hwajung Kang and Amir h. Karamlou and Robert Loredo and David McKay and Alberto Maldonado and Antonio Macaluso and Antonio Mezzacapo and Zlatko Minev and Ramis Movassagh and Giacomo Nannicini and Paul Nation and Anna Phan and Marco Pistoia and Arthur Rattew and Joachim Schaefer and Javad Shabani and John Smolin and John Stenger and Kristan Temme and Madeleine Tod and Ellinor Wanzambi and Stephen Wood and James Wootton.
```

Comment:
1. good resource
2. looks like Qiskit is a python library. Should it be under Python?
3. Or maybe we need a new category in the subjects list for quantum computing
4. It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.
5. Use et al. for long contributor lists

Reply:
ok will add it under python.